master-protected-branches/master.cfg

```diff
@@ -151,7 +151,7 @@ for w_name in ['hz-bbw']:
         addWorker(w_name, i, '-debian-10','quay.io/mariadb-foundation/bb-worker:debian10', jobs=jobs, save_packages=True)
         addWorker(w_name, i, '-debian-10-debug-embed', 'quay.io/mariadb-foundation/bb-worker:debian10', jobs=14, save_packages=True)
         addWorker(w_name, i, '-debian-11-debug-ps-embed', 'quay.io/mariadb-foundation/bb-worker:debian11', jobs=14, save_packages=True)
-        addWorker(w_name, i, '-fedora-35', 'quay.io/mariadb-foundation/bb-worker:fedora35', jobs=jobs, save_packages=True)
+        addWorker(w_name, i, '-fedora-36', 'quay.io/mariadb-foundation/bb-worker:fedora36', jobs=jobs, save_packages=True)
```

Comment:
this needs to be removed if the builder is deleted below or the builder needs to be changed from 35 to 36 if we want to keep fedora36 as protected branches